_drafts/SC3-1-1-html-lang-valid.md

```diff
@@ -0,0 +1,44 @@
+---
+name: SC3-1-1-html-lang-valid
+
+description:
+ This rule checks the `xml:lang` and/or the `lang` attribute, and that it conforms to [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt).
+
+success_criterion:
+- 3.1.1
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Annika Nietzio
+- Jey Nandakumar
+---
+
+## Test Procedure
+
+### Applicability
+
+The root element of the page, if it is an `html` element with a `lang` and/or `xml:lang` attribute.
+
+### Expectation
+
+The value of the `lang` or `xml:lang` attribute conforms to one of the values from the list - [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt).
```

Comment:
This sentence could be clearer. BCP47 is not a list, but a document describing a "best current practice" for constructing language identifiers. IANA, on the other hand, maintains an authoritative list of tags conforming to BCP 47. Maybe something along the lines of:

> The value of the `lang` or `xml:lang` attribute conforms to [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt).

One might also add a short description of what BCP 47 actually is.

Reply:
Thanks @kasperisager